udata_search_service/presentation/api.py

```diff
@@ -346,6 +367,78 @@ def reuse_unindex(reuse_id: str, reuse_service: ReuseService = Provide[Container
     abort(404, 'reuse not found')
 
 
+@bp.route("/dataservices/", methods=["GET"], endpoint='dataservice_search')
+@inject
+def dataservices_search(dataservice_service: DataserviceService = Provide[Container.dataservice_service]):
+    try:
+        request_args = DataserviceArgs(**request.args)
+    except ValidationError as e:
+        abort(400, e)
+
+    results, results_number, total_pages = dataservice_service.search(request_args.dict())
+
+    next_url = url_for('api.dataservice_search', q=request_args.q, page=request_args.page + 1,
+                       page_size=request_args.page_size, _external=True)
+    prev_url = url_for('api.dataservice_search', q=request_args.q, page=request_args.page - 1,
```

Comment:
Nit: should this be clamped to make sure we never get a `page=0` or worse, a negative one?

Reply:
I think edge cases are dealth in [make_response](https://github.com/opendatateam/udata-search-service/blob/04b0be7d247eebb1b44e322fb39af183cf9bec55/udata_search_service/presentation/api.py#L96)